app/src/main/java/org/oppia/app/help/faq/FAQCategoryAdapter.kt

```diff
@@ -0,0 +1,68 @@
+package org.oppia.app.help.faq
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.NonNull
+import androidx.recyclerview.widget.RecyclerView
+import org.oppia.app.R
+import org.oppia.app.databinding.FAQItemBinding
+
+/** The Recycler View adapter in the [FAQFragment]. */
+class FAQCategoryAdapter(
+  private val arrayList: ArrayList<FAQViewModel>
+) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+
+  private val header = 1
```

Comment:
Create enum instead of this. Check enum in `StoryFragmentPresenter`.

Reply:
Done